src/Stratis.Bitcoin.Features.Api/ApiFeature.cs

```diff
@@ -124,6 +125,7 @@ public static IFullNodeBuilder UseApi(this IFullNodeBuilder fullNodeBuilder, Act
                         services.AddSingleton(fullNodeBuilder);
                         services.AddSingleton(options);
                         services.AddSingleton<ApiSettings>(new ApiSettings(setup));
+                        services.AddSingleton<DashboardController>();
```

Comment:
I don't think this controller should be included in this feature as we don't want to have the Dashboard enabled by default.
ApiFeature is the feature enabling APIs, and I think the dashboard should be a Feature in itself.

Reply:
Yes thats true, and it will be a feature that depends on the api feature.